### DISCUSSION TITLE
XAML to SVG

### DISCUSSION THREAD

**Discussion by Shujee** [ElinamLLC/SharpVectors#197] [Q&A]
Is there a way to save/export loaded SVG DOM back to SVG format?

I'm basically working on a (very) simple SVG editor. This is WPF desktop application on top of .NET 5. So I intend to support basic SVG components (lines, rectangles, ellipses, shapes and text) with basic fill and stroke. The app will open SVG files (that were created using this app, so we do not need full-blown SVG support, only the above elements). 

I guess I can load SVG content into the application using `SvgCanvas` class (though I haven't tried it yet). I can then allow user to do operations on it (standard transforms, changing fills/strokes etc.) and then let them save it back to SVG. It's this last step that I'm not sure how to tackle. Any help will be appreciated.

**Comment by paulushub** [comment]
@Shujee Please take a look at the article and code link below. It is an editor implemented in GDI+ that saves to SVG format.
[SVG Artiste - An SVG Editor](https://www.codeproject.com/Articles/71151/SVG-Artiste-An-SVG-Editor)
